public/app/services/searchService.js

```diff
@@ -19,7 +19,7 @@
 
                 return function (parameters, callback) {
                     $.ajax(_.merge({}, ajaxOptions, {
-                        url: baseUrl + '?q=' + queryString(parameters || {}),
+                        url: baseUrl + '?q=' + queryString(parameters || {}) + (noCache ? '&noCache=1' : ''),
```

Comment:
I know the argument is moot, but isn't there a way to pass a map of parameters to the GET rather than manually creating the query string?